src/lib/components/select/namespace-select.svelte

```diff
@@ -7,9 +7,9 @@
   import { getAppContext } from '$lib/utilities/get-context';
   import { isCloud } from '$lib/utilities/env';
 
-  $: namespaces = (getAppContext('namespaces') ?? []).map(
-    (namespace) => namespace.namespaceInfo.name,
-  );
+  $: namespaces = (getAppContext('namespaces') ?? [])
+    .map((namespace) => namespace?.namespaceInfo?.name ?? undefined)
```

Comment:
The `?? undefined` seems extraneous?

Reply:
It makes TS happy because that type is `string | null | undefined` so setting that to undefined and then filtering appeases the compiler